source/administration/upgrade.md

```diff
@@ -11,19 +11,16 @@ To start, select one of the following guides:
 
 ## Upgrade Team Edition to 3.1.x and later
 
-**Important Note:** Security-related changes were made in 3.8 that require you to verify settings in the System Console before upgrading from version 3.7.x and earlier to any version greater than 3.8.0.
-
-**To prepare your system when upgrading from 3.7.x and earlier**:
-
-  1. In the System Console, go to **General > Configuration** and make sure that the **Site URL** is specified. It must not be empty.
-  2. In the System Console, go to **General > Logging** and make sure that the **File Log Directory** field is either empty or has a directory path only. It must not have a filename as part of the path.
+**Important Note:** Security-related changes were made in `v3.6.x` and later that require you to specify the Site URL setting in the System Console before upgrading from version `v3.5.x` and earlier to any version greater than `v3.6.x`. In the System Console, go to **General > Configuration** and make sure that the **Site URL** is specified. It must not be empty.
```

Comment:
I wonder if we could get Jeff's help make this easier to understand.

An IT admin reading it for the first time may not understand what this means. 

Maybe we can just say that Site URL is now required if you upgrade to 3.6.X and later? And that it can be set in config.json or System Console (with a link to the config setting doc).

Also, might be worth formatting as a "note" to catch IT admin's attention, e.g.

![image](https://cloud.githubusercontent.com/assets/13119842/25505988/4fcba126-2b72-11e7-9901-ff47ad770d0c.png)